doc/user_manual/postprocessor.tex

```diff
@@ -1281,6 +1280,22 @@ \subsubsection{Interfaced}
 of equal size to \xmlNode{outputLen} might have unexpected results.  In general, we recommend using a
 \xmlNode{subseqLen} that is roughly an order of magnitude smaller than \xmlNode{outputLen}.
 
+\paragraph{Method: dataObjectLabelFilter}
+This Post-Processor allows to filter the portion of a dataObject, either PointSet or HistorySet, with a given clustering label.
+A clustering algorithm associate a unique cluster label to each element of the dataObject (PointSet or HistorySet). 
+This cluster label is a natural number ranging from $0$ to $N$ where $N$ is the number of obtained clusters. 
+Recall that some clustering algorithms (e.g., K-Means) receive $N$ as input while others (e.g., Mean-Shift) determine $N$ after clustering has been performed. 
+Thus, this Post-Processor is naturally employed after a data-mining clustering techniques has been performed on a dataObject so that each clusters 
+can be analyzed separately.
+
+In the \xmlNode{PostProcessor} input block, the following XML sub-nodes are required,
+independent of the \xmlAttr{subType} specified:
+
+\begin{itemize}
+   \item \xmlNode{clusterIDs}, \xmlDesc{string, required field}, ID of the clustering label
+   \item \xmlNode{label}, \xmlDesc{integers, required field}, IDs of the chosen clustering labels. Note that more than one label can be provided as input
```

Comment:
What is the difference between "clusterIDs" and "label"? 

PS. Is it "label" (singular) or "labels" (plural)?